### PR TITLE
simplify switches, add --raw

### PR DESCRIPTION
**Pull request by slingamn**

### CODE REVIEW

File: ircdog.go [hunk 1]
```diff
@@ -86,10 +86,10 @@ Options:
 	--tls-noverify      Don't verify the provided TLS certificates.
 	--listen=<address>  Listen on an address like ":7778", pass through traffic.
 	--hide=<messages>   Comma-separated list of commands/numerics to not print.
-	--no-italics        Don't use the ANSI italics code to represent italics.
-	--no-controls       Don't use the control character escapes.
+	-r --raw            Don't interpret IRC control codes.
+	--escape            Display incoming lines with irc-go escapes.
```
Comment (DanielOaks) [inline ircdog.go:90]:
```suggestion
	-e --escape         Display incoming lines with irc-go escapes.
```

File: ircdog.go [hunk 2]
```diff
@@ -86,10 +86,10 @@ Options:
...
+	--italics           Enable ANSI italics codes (not widely supported).
```
Comment (DanielOaks) [inline ircdog.go:91]:
```suggestion
	-i --italics        Render italic text with ANSI (not widely supported).
```

**Comment by DanielOaks** [comment]
Nice, I don't mind these new switches. Some shorthand would be nice, and I like things being enabled by default but doing this with the italics switch makes sense. Good job!

**Review by DanielOaks — COMMENTED** [review]
Nice, I don't mind these new switches. Some shorthand would be nice, and I like things being enabled by default but doing this with the italics switch makes sense. Good job!

File: ircdog.go [hunk 3]
```diff
@@ -86,10 +86,10 @@ Options:
 	--tls-noverify      Don't verify the provided TLS certificates.
 	--listen=<address>  Listen on an address like ":7778", pass through traffic.
 	--hide=<messages>   Comma-separated list of commands/numerics to not print.
-	--no-italics        Don't use the ANSI italics code to represent italics.
-	--no-controls       Don't use the control character escapes.
+	-r --raw            Don't interpret IRC control codes.
```
Comment (DanielOaks) [inline ircdog.go]:
```suggestion
	-r --raw            Don't interpret IRC control codes when sending or receiving lines.
```

The text above this references the `--no-controls` option, which has been removed in favour of `--raw`. This line should also clarify that it applies to incoming **and** outgoing text.

The above text also says this, which needs to be updated to specify the `-e` escape (if the `-e` shorthand is allowed):
```
When the -r option is used, lines are displayed with the irc-go escapes
```

**Comment by DanielOaks** [comment]
**edit:** noticed a couple of cases where text should be updated for clarity/accuracy with the new changes, whoo!

**Comment by slingamn** [comment]
I think actually the lack of short forms for `--escape` and `--italics` is intentional: it's a hint that if you don't know exactly what they do, you probably don't want them.

**Pull request merged**